site/assets/js/application.js

```diff
@@ -118,13 +118,15 @@
   }
 
   // Insert copy to clipboard button before .highlight
-  var btnHtml = '<div class="bd-clipboard"><button type="button" class="btn-clipboard" title="Copy to clipboard">Copy</button></div>'
+  var btnHtml = '<div class="bd-clipboard"><button type="button" class="btn-clipboard" title="Copy to clipboard">Copy</button></div>' +
+    '<div class="bd-edit"><button type="button" class="btn-edit" title="Edit on StackBlitz">Try it</button></div>'
```

Comment:
```suggestion
    '<div class="bd-edit"><button type="button" class="btn-edit" title="Try it on StackBlitz">Try it</button></div>'
```

`title` needs to contain visible label to ensure accessibility for e.g. voice control.